crates/goose/src/agents/agent.rs

```diff
@@ -1210,6 +1215,14 @@ impl Agent {
 
                 tokio::task::yield_now().await;
             }
+
+            // Wait for naming task to complete if it was spawned
+            // We don't propagate panics since session naming is a low-value background task
```

Comment:
I added this to satisfy the bot even though it isn't typical in this project to spam the user on things that don't matter. open to advice from a human on it

Reply:
I disagree with the bot. these warning just go to the log, we won't be looking into it.